### DISCUSSION TITLE
.NET 4.8 and Azure App Service

### DISCUSSION THREAD

**Discussion by SkyeHoefling** [dnnsoftware/Dnn.Platform#3856] [General] [answered]
Azure App Service is going to be automatically upgrading all servers to .NET Framework 4.8. As far as I know DNN currently support .NET 4.7.2 and there aren't many changes between the 2 versions. 

* Does anyone have experience running .NET 4.8 and the current releases of DNN? 
* Should there be concern about this switch over?
* Are we going to make DNN explicitly target .NET 4.8 in DNN 10?

The switch over in Azure will start July 14, 2020 and all legacy resources will automatically be updated September 15th, 2020. Azure is recommending that we make sure sites are compatible before July 14th.

**Comment by mitchelsellers** [comment, marked as answer]
I have a number of installations running on .NET 4.8 and there appear to be no issues at all.  We ARE planning to bump to 4.8 for 10.x due to the Same-Site Cookie Behavior changes that are included.

So I think we are good for now anyway
